### PR TITLE
Rafbranch

### PR DESCRIPTION
**Pull request by RafaelCenzano**

### CODE REVIEW

**Comment by RafaelCenzano** [comment]
Please specify where <div style"clear:both"> this needs to go

File: lowelldevclub/templates/about.html [hunk 1]
```diff
@@ -7,44 +8,55 @@
...
+  				<p>Python, Javascript, Hyper Text Markup Language, Cascading Style Sheets</p>
+  				<div style="margin: 24px 0;">
+    				<a href="#" target="_blank"><i class="fa fa-github"></i></a>
+						<a href="#" target="_blank"><i class="fa fa-linkedin"></i></a>
+    				<a href="#" target="_blank"><i class="fa fa-facebook"></i></a>
+    				<a href="#" target="_blank"><i class="fa fa-instagram"></i></a>
+  				</div>
+  				<p><button>Contact</button></p>
+				</div>
+			</div>
```
Comment (calee14) [inline lowelldevclub/templates/about.html]: The <div style="clear: both"></div> right in between line 58 and 59.

Comment (calee14) [inline lowelldevclub/templates/about.html]: Add the <div style="clear: both"></div> right in between line 58 and 59.

Comment (RafaelCenzano) [reply lowelldevclub/templates/about.html]: Fixed the image problem

Comment (RafaelCenzano) [reply lowelldevclub/templates/about.html]: add what between those lines?

**Pull request merged**